polaris-react/src/components/Navigation/components/Section/Section.tsx

```diff
@@ -103,7 +103,7 @@ export function Section({
 
   const sectionHeadingMarkup = title && (
     <li className={styles.SectionHeading}>
-      <Text as="span" variant="headingXs" color="subdued">
+      <Text as="span" variant="bodySm" fontWeight="medium" color="subdued">
```

Comment:
Does this need to be beta-flagged?

Reply:
Visually, it stays the same in gen2 because `bodySm` and `headingXs` are the same size, so it should have no impact.